docker-compose.yaml

```diff
@@ -4,6 +4,7 @@ services:
   homing-pigeon:
     volumes:
     - ./sockets:/tmp
+    - ./:/go/src/github.com/softonic/homing-pigeon
```

Comment:
Is this needed?

Reply:
For develop it's useful, how do you test the source code or launch the tests via Docker?

Reply:
this project has gomod,mounting it in the gopath should not be needed. I take it you got the path from somewhere (Dockerfile), maybe we should change there too.

I find the docker workflow slowing down a lot. I use my host for dev-test cycles, and test the bin in docker. Otherwise build time can increase (although there are tricks to avoid that)